decision/index-en.md

```diff
@@ -71,6 +71,7 @@ The following design decision can be obseleted if you find they are not valid an
 				<li>Github related issue <a href="https://github.com/wet-boew/wet-boew/issues/7314">wet-boew/wet-boew#7314</a></li> 
 			</ul>
 		</li>
+		<li>http://wet-boew.github.io/wet-boew-documentation/decision/index-en.html</li>
```

Comment:
You are missing the anchor ```<a>``` element and the title of your design decision draft.

This link should be pointing to your new created file. Once published, the *.md file are compiled into *.html file.  The file name after publication will be: draft-2018-5.html